GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -495,6 +495,9 @@ void Renderer::renderRenderData(RenderData* render_data,
         if (!render_data->alpha_blend()) {
             glDisable (GL_BLEND);
         }
+       if( !render_data->alpha_to_coverage()) {
+           glDisable(GL_SAMPLE_ALPHA_TO_COVERAGE);
+        }
```

Comment:
If prev state was set as disable for GL_SAMPLE_ALPHA_TO_COVERAGE how do we enable it when the flag is true? We depend on previous call to RenderRenderData to set the state properly. Prefer to explicitly enable/disable state.